### PR TITLE
The Gun Sound Overhaul

### PR DESCRIPTION
**Pull request by XSlayer300**
## About The Pull Request
Reoverhauls almost every gun sound in the game, replacing every sound that was from CM or making the sounds from other games less noticeable.  This also adds unique sounds to the dropship weapons, as well replacing the empty gun sound with a new one.

All sounds are from (if I remember correctly):

- Call of Duty series

- Halo series

- Insurgency

- Counter Strike: Global Offensive

- Team Fortress 2

- Ghost in the Shell Stand Alone Complex - First Assault Online

- Fallout series

- And others that I forgot.

- Along with some of the original CM sounds.

**All (new) sounds are from their respective owners, and I (and the entire community) do not take ownership from these sounds, mixed or not. They use the CC-BY-NC-SA 3.0/Creative Commons Attribution-NonCommercial-ShareAlike 3.0 United States license.**

**If any IP wants me to remove those sounds mixed in with the original ones, I would be glad to oblige.**

## Why It's Good For The Game
Because for me, I'm getting sort of tired hearing CM sounds in TGMC, so we're bringing something unique.

## Changelog
:cl:
tweak: Completely OVERHAULS almost all of the useable weapons, adding new ones...
del: ... and removing unused sounds or sounds from the leaked CM codebase.
tweak: All sounds now have consistent volume. Not all weapon sounds are changed.
tweak: The gunship's weapons now use a unique sound when firing
tweak: Changed the empty reloading sound.
tweak: Changed the *salute emote.
tweak: If you like/love the sounds, great. But if you don't like it, I respect your opinion.
tweak: All sounds use the CC-BY-NC-SA 3.0/Creative Commons Attribution-NonCommercial-ShareAlike 3.0 United States license. See PR for list of sounds used.
tweak: Fixed minor typos.
/:cl:

### CODE REVIEW

**Comment by XSlayer300** [comment]
Switched current license to **CC-BY-NC-SA 3.0/Creative Commons Attribution-NonCommercial-ShareAlike 3.0 United States**

**Comment by XSlayer300** [comment]
Awaiting review...

**Review by TiviPlus — APPROVED** [review]
Ok fuck it its good enough

File: code/game/objects/machinery/smartgun_mount.dm [hunk 1]
```diff
@@ -43,8 +43,8 @@
 
 // The actual gun itself.
 /obj/item/standard_hmg
-	name = "\improper TL-102 Mounted Heavy Smartgun"
-	desc = "The TL-102 Heavy Machinegun. IFF capable. No extra work required, just deploy it."
+	name = "\improper TL-102 mounted heavy smartgun"
+	desc = "The TL-102 heavy machinegun, it's too heavy to be carried to be operated without the tripod. IFF capable. No extra work required, just deploy it."
```
Comment (Hughgent) [inline code/game/objects/machinery/smartgun_mount.dm:47]:
```suggestion
	desc = "The TL-102 heavy machinegun, it's too heavy to be carried or to be operated without the tripod. IFF capable. No extra work required, just deploy it."
```

**Review by Hughgent — APPROVED** [review]

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**